src/main/java/seedu/address/model/coursemate/Rating.java

```diff
@@ -31,6 +33,40 @@ public static boolean isValidRating(String test) {
         return test.matches(VALIDATION_REGEX);
     }
 
+    /**
+     * Returns a rating text shown by number of stars according to the rating.
+     * If rating is 0, empty text is returned.
+     * @param rating string of integer between 1 and 5.
+     */
+    public static String toStringWithStars(String rating) {
+        requireNonNull(rating);
+        checkArgument(isValidRating(rating), MESSAGE_CONSTRAINTS);
+
+        int ratingValue;
+        try {
+            ratingValue = Integer.parseInt(rating);
+        } catch (NumberFormatException e) {
+            // when rating is a non-integer string (though this would not happen)
+            ratingValue = 0;
+        }
+
+        if (0 < ratingValue && ratingValue < 6) {
```

Comment:
`ratingValue <= 5` is more intuitive compared to `ratingValue < 6` since it gives the maximum acceptable value.

BTW given the rating text is always `<= 5`, you should use assertions instead of putting it into the control flow.

Reply:
Got it